source/Calamari/Kubernetes/SetupKubectlAuthentication.cs

```diff
@@ -45,7 +45,7 @@ public class SetupKubectlAuthentication
             this.workingDirectory = workingDirectory;
         }
 
-        public CommandResult Execute(string accountType)
+        public CommandResult Execute(string accountType, Kubectl kubectl = null)
```

Comment:
Whats the reason that sometimes we can pass through the kubectl (based on where its called in the  `KubernetesAuthContextConvention` presumably via the container) and in the other case (via `KubernetesContextScriptWrapper`) we dont?

Reply:
Yep - basically allowing the wrapper to continue to function until we can remove it (once all kubectl script steps are moved to calamari commands).

Reply:
I've managed to remove this and put it as a constructor dependency. It's not the wrappers responsibility to create the kubectl instance which makes more sense and will go away once we can delete the wrapper.